core/src/components/modal/modal.tsx

```diff
@@ -560,6 +582,43 @@ export class Modal implements ComponentInterface, OverlayInterface {
     return eventMethod(this.el, 'ionModalWillDismiss');
   }
 
+  /**
+   * Move a sheet style modal to a specific breakpoint.
+   */
+  @Method()
+  async setBreakpoint(breakpoint: number): Promise<void> {
```

Comment:
How do you feel about renaming this public API to `setActiveBreakpoint`/`setCurrentBreakpoint`? It would be more explicit that it's effecting the current breakpoint.

Reply:
Or `moveToBreakpoint`? what do you like most?

Reply:
Agreed, let's go with `setCurrentBreakpoint`. Also aligns with the internal behaviors of the function.